rn/Teacher/src/modules/submissions/list/submission-entities-reducer.js

```diff
@@ -41,7 +41,7 @@ export const submissions: Reducer<SubmissionsState, any> = handleActions({
             selectedAttachmentIndex: 0,
           },
         }), {})
-      return { ...state, ...incoming }
+      return incoming
```

Comment:
When we refresh we don't want to keep around all the old submissions as well. In this case the submissions needed to be grouped but at some point they were being requested ungrouped and since we would just "update" the existing models and leave around the others we were getting too many submissions in speed grader". Instead since we are "refreshing" it's reasonable to assume that we would replace all contents.